src/staticanalysis/bot/static_analysis_bot/__init__.py

```diff
@@ -121,6 +121,20 @@ class Reliability(enum.Enum):
     Medium = 'medium'
     Low = 'low'
 
+    def invert(self):
+        '''
+        Verbalizez the opposite of `value` of reliability to be used in coherent
+        sentences.
+        '''
+        if self.value == 'high':
```

Comment:
It would be cleaner to use a dict of inverted values here:

```
inversions = {
  'high': 'low',
 ...
}
return inversions.get(self.value, 'unknown')
```